content/docs/lifting-state-up.md

```diff
@@ -130,9 +130,9 @@ function toFahrenheit(celsius) {
 }
 ```
 
-These two functions convert numbers. We will write another function that takes a string `temperature` and a converter function as arguments and returns a string. We will use it to calculate the value of one input based on the other input.
+Dua fungsi ini mengkonversi angka. Kami akan menulis fungsi lain yang menggunakan fungsi *string* `temperature` dan fungsi konverter sebagai argumen dan mengembalikan string. Kami akan menggunakannya untuk menghitung nilai dari suatu input bersarakan input lainnya.
```

Comment:
`We will write another function...` -> `Kita akan menulis fungsi...`

Reply:
- `Dua fungsi ini mengkonversi angka.` -> `Dua fungsi ini dapat mengkonversi angka.` karena makna tersebut lebih cocok jika disandingkan dengan kalimat aslinya.
- ``Kami akan menulis fungsi lain yang menggunakan fungsi *string* `temperature` dan fungsi konverter sebagai argumen dan mengembalikan string.``
-> ``Kita akan menulis fungsi lain yang menggunakan _string_ `temperature` dan fungsi konverter sebagai argumen dan menghasilkan sebuah _string_``